src/main/java/bio/terra/service/dataset/DatasetStorageAccountDao.java

```diff
@@ -0,0 +1,174 @@
+package bio.terra.service.dataset;
+
+import bio.terra.common.exception.RetryQueryException;
+import bio.terra.service.snapshot.exception.CorruptMetadataException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.DataAccessException;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Isolation;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.UUID;
+
+import static bio.terra.common.DaoUtils.retryQuery;
+
+/**
```

Comment:
This isn't a javadoc comment so it shouldn't start with `/**`
```suggestion
/*
```